submissions/omolayoOdupe/html-form/contactForm.html

```diff
@@ -0,0 +1,84 @@
+<!DOCTYPE html>
```

Comment:
Kindly work on the existing contact page you have. Make every change there. You don't need to create a new page

Reply:
@adaugochi 
It is the existing Contact Page.

I removed some detail, added some other details and resaved.

It is the existing contactpage. I just resaved it after editing.

Reply:
ok